hugo/content/blog/2020-12-24-mumble-1-4-0-snapshot.md

```diff
@@ -0,0 +1,277 @@
+---
+title: First Mumble 1.4.0 Development Snapshot
+author: Robert Adam
+date: 2020-12-24
+categories:
+  - News
+  - Development Snapshot
+---
+
+This year we have a Christmas present for you: We are proud to announce that the first **snapshot release of Mumble 1.4.0** is released as of now. It
```

Comment:
```suggestion
This year we have a Christmas present for you: we are proud to announce that the first **snapshot release of Mumble 1.4.0** is released as of now. It
```

Reply:
Afaik it's correct the way it is right now :thinking:

Reply:
It actually depends: https://www.grammarly.com/blog/capitalization-after-colons

Reply:
Guess I always did it wrong then xD